schedule/qam/common/security/selinux.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: selinux
+conditional_schedule:
+  versions:
+    VERSION:
+      "15-SP2":
+      - security/selinux/semanage_boolean
```

Comment:
Why are you changing order of tests ?

Reply:
There is a "boot" in semanage_boolean.  The booting generates a lot audit log (I guess that openqa worker sets up sut before it runs tests). These logs takes "audit2allow  -w -i  auditlog"  in audit2allow test a lot time to complete it outputting. It causes the script_output_validation timeout. So I make semanage_boolean run after audit2allow on sle12sp5.

Reply:
Why only on sle12sp5 ?

Reply:
This set of selinux test  can only work well with sle15sp2 and sel12sp5. Many tests will break with earlier versions. It will take a lot work to fix/work round them. The code change would be huge. Plus the earlier versions are already ltss. So I would just like to schedule the selinux test on sle15sp2 and sle12sp5. 
In addition sle15 and sle12 use different selinux policy. That could get access denied on one system but accepted on the other. 
semanage_boolean is fine running before audit2allow on sle 15sp3 and sp2.   On sle12sp5 it has to run after audit2allow.